### PR TITLE
replace `SolverTools` by `SolverCore`

### PR DESCRIPTION
**Pull request by tmigot**

### CODE REVIEW

**Review by dpo — APPROVED** [review]

**Comment by abelsiqueira** [comment]
Should be rebased after #13, adding compat bounds.

File: Project.toml [hunk 1]
```diff
@@ -17,4 +17,5 @@ NLPModelsJuMP = "792afdf1-32c1-5681-94e0-d7bf7a5df49e"
 NodeJS = "2bd173c7-0d6d-553b-b6af-13a54713934c"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 SolverBenchmark = "581a75fa-a23a-52d0-a590-d6201de2218a"
+SolverCore = "ff4d7338-4cf1-434d-91df-b86cb86fb843"
```
Comment (abelsiqueira) [inline Project.toml:20]: It also needs a compat bound, and please also add it to `utils.jl` to a list of JSO packages, please.

Comment (abelsiqueira) [reply Project.toml:20]: I will do it directly to main - since we don't have a test for this yet.

**Comment by abelsiqueira** [comment]
Thank you

**Pull request merged**